Makefile

```diff
@@ -27,7 +27,10 @@ BOWER=bower_components
 JS=js
 
 # Pure ERMrest API
-SOURCE=$(JS)/core.js \
+SOURCE=markdown-it/markdown-it.min.js \
```

Comment:
Added the markdown-it parser in `SOURCE` for now. I'm unsure about how plugins should be situated within ermrestjs' structure. (Maybe a `plugins` or `lib` or `vendor` folder at the same level as the `js` folder?) We can revisit this file structure if/when more third party tools are needed in ermrestjs.